src/Video.js

```diff
@@ -73,6 +73,7 @@ export default class Video extends React.Component {
     captionsHidden: true,
     videoRefreshing: false,
     showControls: true,
+    isControlVisible: true,
```

Comment:
do we need another var for this? we already have showControls

Reply:
Yes, it serves a different purpose. It is used to hide controls when the opacity value is animated to 0. because TouchableOpacity is still clickable when the opacity value is animated to 0.